src/System.Drawing.Common/src/System/Drawing/NativeStructs.Unix.cs

```diff
@@ -54,7 +54,6 @@ internal struct LOGFONT
         internal string lfFaceName;
     }
 
-    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
```

Comment:
I would keep `[StructLayout(LayoutKind.Sequential)]` without the CharSet to match the style used for other structs in this file.